resources/assets/js/stories/Modal.stories.tsx

```diff
@@ -113,8 +113,8 @@ stories.add(
             </div>
             <form data-c-padding="normal">
               <Input
-                htmlId="modal-input-1"
-                formName="modal-input-1"
+                id="modal-input-1"
+                name="modal-input-1"
                 label="This is a text input"
                 required={false}
```

Comment:
This is more of a general note, but any boolean props can be passed as `required` for `true` or leave it out for `false`. You don't need to explicitly pass `prop={false}` or `prop={true}`.

Reply:
Yup haha, will clean that up.